vllm/config.py

```diff
@@ -2658,6 +2658,16 @@ class SpeculativeConfig:
         ParallelConfig] = None  # type: ignore
     """The parallel configuration for the draft model initialized internal."""
 
+    # params for relaxed thinking
+    posterior_alpha: Optional[float] = 1.0
+    """The alpha value for the posterior distribution in speculative decoding.
+    Used when relaxed thinking is enabled."""
+    relaxed_thinking: Optional[bool] = False
```

Comment:
If `None` is not a valid value, the type should not be `Optional`. Please also propagate this to other places where these parameters are not `None`

```suggestion
    posterior_alpha: float = 1.0
    """The alpha value for the posterior distribution in speculative decoding.
    Used when relaxed thinking is enabled."""
    relaxed_thinking: bool = False
```

Reply:
Done.